style.css

```diff
@@ -0,0 +1,194 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* Basic resets */
```

Comment:
kindly remove all the comments from all the files:
![image](https://user-images.githubusercontent.com/34653764/83825179-7b48d300-a6d0-11ea-852c-3d54ac4bc2ff.png)